curriculum/challenges/english/25-front-end-development/workshop-cat-painting/646f507e4d1cd323f17db4fc.md

```diff
@@ -15,25 +15,25 @@ Use class selector to target the `.cat-whisker-right-bottom` element, then move
 You should have a `.cat-whisker-right-bottom` selector.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.cat-whisker-right-bottom'))
+assert.exists(new __helpers.CSSHelp(document).getStyle('.cat-whisker-right-bottom'));
 ```
 
 Your `.cat-whisker-right-bottom` selector should have a `position` property set to `absolute`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.cat-whisker-right-bottom')?.position === 'absolute')
+assert.equal(new __helpers.CSSHelp(document).getStyle('.cat-whisker-right-bottom')?.position,'absolute');
```

Comment:
```suggestion
assert.equal(new __helpers.CSSHelp(document).getStyle('.cat-whisker-right-bottom')?.position, 'absolute');
```
For style consistency please also make sure there's space after the comma in changed lines.